src/file.rs

```diff
@@ -1,13 +1,15 @@
-use crate::log::{Log, LogFormat};
-use parking_lot::Mutex;
 use std::{
   fs::File,
   io::{self, BufWriter, Write},
   path::{Path, PathBuf},
   sync::atomic::{AtomicUsize, Ordering},
 };
+
+use parking_lot::Mutex;
 use time::{Duration, OffsetDateTime, Time};
 
+use crate::log::{Log, LogFormat};
+
```

Comment:
For imports, I usually just let rust-analyzer do its thing, but I don't mind this

Reply:
If you use code actions to import missing items, this is how rust-analyzer will do things. But it won't enforce it if you merge all the blocks together.